openapi/googleapis.books/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to Google Books from Ballerina
```

Comment:
This should change as [Google Books API {version}](link)?
Example `Connects to [Google Docs API v1.0](https://developers.google.com/docs/api) from Ballerina
`

Reply:
Resolved